### ISSUE TITLE
💥 Crash on `-[PendoAPI initWithSession]`

### ISSUE THREAD

**Issue by msrutek-paylocity** [pendo-io/pendo-mobile-sdk#184]
## **Platform + Version**
iPadOS 16.7.9, iPad (5th generation) 

## **SDK Version**
3.3.2

## **Framework**
Native, SwiftUI

## **Describe the bug**
Crash of the app.

## **To Reproduce**
Yet unknown.

## **Expected behavior**
No crash.

## **Stack Trace**

```
Crashed: pendo.api.initQueue
EXC_BAD_ACCESS 0x00000001081368d4

0  Pendo                          0x668d4 -[PendoAPI initWithSession] + 10
1  Pendo                          0x5931c __44+[PNDTrackEventsUtils sendTrack:properties:]_block_invoke + 388
2  libdispatch.dylib              0x637a8 _dispatch_call_block_and_release + 24
3  libdispatch.dylib              0x64780 _dispatch_client_callout + 16
4  libdispatch.dylib              0xa298 _dispatch_lane_serial_drain$VARIANT$mp + 644
5  libdispatch.dylib              0xad70 _dispatch_lane_invoke$VARIANT$mp + 396
6  libdispatch.dylib              0x14dc8 _dispatch_workloop_worker_thread + 632
7  libsystem_pthread.dylib        0x1bd0 _pthread_wqthread + 284
8  libsystem_pthread.dylib        0x1720 start_wqthread + 8
```

## ℹ️

We got a crash that we haven't observed before. So far, a single occurrence.

**Comment by TehilaTaub** [comment]
Hi @msrutek-paylocity 
It is hard to understand from these few lines. Can you please attach the full crash log?

**Comment by msrutek-paylocity** [comment]
Hello @TehilaTaub, unfortunately, this is all the info we have.

**Comment by MikePendo** [comment]
@msrutek-paylocity 
We just released new version 3.4.3, we had fixed significant memory issue that might arise on bad network. That might trigger or be the cause for the memory issue you have, any chance you could fetch the latest release and see and track the strange crashes once again. 
My comment applies to almost all issues you have found that related to memory, so I will copy paste that messages to the the relevant threads

**Comment by TehilaTaub** [comment]
Hi @msrutek-paylocity,
Please upgrade to 3.4.3 and if you notice similar issues, please open a new ticket with the crash logs.
Closing this one now.
